client/components/theme-tier/constants.js

```diff
@@ -5,7 +5,22 @@ import {
 	PLAN_PREMIUM,
 	getPlan,
 } from '@automattic/calypso-products';
-import { translate } from 'i18n-calypso';
+import { englishLocales } from '@automattic/i18n-utils';
+import i18n, { translate } from 'i18n-calypso';
+
+const getIncludedWithLabel = ( planSlug ) => {
+	const localeSlug = i18n.getLocaleSlug();
+
+	const shouldShowNewString =
+		( localeSlug && englishLocales.includes( i18n.getLocaleSlug() ) ) ||
+		i18n.hasTranslation( 'Included with %(planName)s' );
+
+	return shouldShowNewString
+		? translate( 'Included with %(planName)s', {
+				args: { planName: getPlan( planSlug )?.getTitle() },
+		  } )
+		: getPlan( PLAN_PERSONAL )?.getTitle();
```

Comment:
Why are we defaulting to the `Personal Plan`? If another consumer uses this helper method it could return some misleading values.

Reply:
You caught a bug! I had the logic duplicated before I created this helper function and forgot to replace the hardcoded plan slug with the given param.

Reply:
Thanks for fixing this!